easybuild/easyconfigs/b/Boost.Python/Boost.Python-1.66.0-intel-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'EB_Boost'
+
+name = 'Boost.Python'
+version = '1.66.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://boostorg.github.io/python'
+description = """Boost.Python is a C++ library which enables seamless interoperability between C++
+ and the Python programming language."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+toolchainopts = {'pic': True}
```

Comment:
Is this necessary for the bindings?

Reply:
Well, it could be, if the bindings (which are a `.so`) are linked by other applications (e.g. `Yade` or `Trilinos`).

Also, it doesn't hurt?